drivers/hmis_data_quality_tool/app/models/hmis_data_quality_tool/enrollment.rb

```diff
@@ -250,6 +268,9 @@ def report_item_fields_from_enrollment(report_items:, enrollment:, report:) # ru
       report_item.head_of_household_count = hh&.select(&:head_of_household?)&.count || 0
       report_item.household_type = household_type(report_item.household_min_age, report_item.household_max_age)
 
+      report_item.hh_veteran_count = hh.select { |hm| hm.client.veteran? }.count || 0
+      report_item.hoh_veteran = hh&.select { |hm| hm.head_of_household && hm.client.veteran? }&.count&.> 0 || false
```

Comment:
Is this the same?
```suggestion
      report_item.hoh_veteran = hh&.any? { |hm| hm.head_of_household && hm.client.veteran? }
```

Reply:
I believe it is, I'll give it a test run to verify